connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties

```diff
@@ -176,7 +176,7 @@ datasources.data-source.enlistment-trace=Defines if WildFly/IronJacamar should r
 
 datasources.data-source.recovery-username=The user name used for recovery
 datasources.data-source.recovery-password=The password used for recovery
-datasources.data-source.recovery-security-domain=The security domain used for recovery, for PicketBox security domains
+datasources.data-source.recovery-security-domain=Deprecated. The security domain used for recovery, for PicketBox security domains
```

Comment:
To deprecate an attribute:

1) Add a setDeprecated call to the AttributeDefinition builder
2) Add a 'datasources.data-source.recovery-security-domain.deprecated=xxxx' string to provide details. For example:  'datasources.data-source.recovery-security-domain.deprecated=PicketBox security domains are not supported in servers running the current version of the datasources subsystem. Use 'authentication-context' to configure Elytron security.'

Reply:
thanks. fixed.